test/contiguous_vertices.jl

```diff
@@ -59,17 +57,12 @@ function LightGraphs.outneighbors(g::VSafeGraph, v)
 end
 
 function LightGraphs.inneighbors(g::VSafeGraph, v)
-    if has_vertex(g, v)
-        inneighbors(g.g, v)
-    else
-        throw(ArgumentError("$v is not a valid vertex in the graph."))
-    end
+    has_vertex(g, v) && return inneighbors(g.g, v)
+    throw(ArgumentError("$v is not a valid vertex in the graph."))
```

Comment:
actually, let's not throw an error. what happens by default now?
```
julia> g
{10, 20} undirected simple Int64 graph

julia> inneighbors(g, 100)
ERROR: BoundsError: attempt to access 10-element Array{Array{Int64,1},1} at index [100]
```
that's fine.

Reply:
so we just the checking method right?

Reply:
Oh, I see what you're doing here. Yeah, keep this.

Reply:
just removed